tsung/erlang_subst/generator.erl

```diff
@@ -0,0 +1,96 @@
+-module(generator).
+-export([uuid/1]).
+-export([case_json/1]).
+
+uuid({Pid, DynData}) ->
+   {A1,A2,A3} = now(),
+   random:seed(A1, A2, A3),
+   HexList = ["0","1","2","3","4","5","6","7","8","9","a","b","c","d","e","f"],
+   string:join([
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      "-",
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      "-",
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      "-",
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      "-",
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList),
+      lists:nth(trunc(random:uniform(16)),HexList)
+   ], "").
+
+
+case_json({Pid, DynData}) ->
```

Comment:
ideally this should read from a file and randomly select between larger and smaller cases